SourceLink and Symbol Packages - are they working for you?

https://github.com/akkadotnet/akka.net/pull/3953 - @mmisztal1980 working on a repro spec now.

For reasons unknown, source link was letting me see where the error was happening. It may be set up incorrectly or our symbol package publishing may not be working correctly.

_Originally posted by @Aaronontheweb in https://github.com/akkadotnet/akka.net/issues/3943#issuecomment-538531518_

@akoker so, we haven't been publishing PDBs correctly then by the sound of it? IIRC we do publish separate symbol packages, but we haven't been pushing those correctly. 

From our 1.3.15 release build logs:

```

2019-09-23T08:24:05.9254751Z 
2019-09-23T08:24:07.2654719Z   Successfully created package 'D:\a\1\s\bin\nuget\Akka.TestKit.1.3.15.nupkg'.
2019-09-23T08:24:07.2775640Z C:\Program Files\dotnet\sdk\2.2.105\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(202,5): warning NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead. [D:\a\1\s\src\core\Akka.TestKit\Akka.TestKit.csproj]
2019-09-23T08:24:07.2869748Z   Successfully created package 'D:\a\1\s\bin\nuget\Akka.TestKit.1.3.15.snupkg'.
2019-09-23T08:24:07.3194336Z Finished Target: CreateNuget
2019-09-23T08:24:07.3195734Z Starting Target: PublishNuget (==> CreateNuget)
2019-09-23T08:24:07.3219726Z Pushing nuget packages
2019-09-23T08:24:07.3294704Z Pushing D:\a\1\s\bin\nuget\Akka.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:08.1817924Z Pushing Akka.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:08.2442348Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:09.4861410Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:09.4870699Z   Created https://www.nuget.org/api/v2/package/ 1242ms
2019-09-23T08:24:09.4895031Z Your package was pushed.
2019-09-23T08:24:09.5087826Z Pushing D:\a\1\s\bin\nuget\Akka.Cluster.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:10.1367820Z Pushing Akka.Cluster.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:10.2003894Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:11.1003227Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:11.1012283Z   Created https://www.nuget.org/api/v2/package/ 900ms
2019-09-23T08:24:11.1028403Z Your package was pushed.
2019-09-23T08:24:11.1208685Z Pushing D:\a\1\s\bin\nuget\Akka.Cluster.Sharding.1.3.15-beta.nupkg Attempts left: 3
2019-09-23T08:24:11.7637442Z Pushing Akka.Cluster.Sharding.1.3.15-beta.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:11.8157652Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:12.7355429Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:12.7367497Z   Created https://www.nuget.org/api/v2/package/ 920ms
2019-09-23T08:24:12.7387686Z Your package was pushed.
2019-09-23T08:24:12.7652349Z Pushing D:\a\1\s\bin\nuget\Akka.Cluster.TestKit.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:13.3857275Z Pushing Akka.Cluster.TestKit.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:13.4460637Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:14.0706637Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:14.0715765Z   Created https://www.nuget.org/api/v2/package/ 625ms
2019-09-23T08:24:14.0731663Z Your package was pushed.
2019-09-23T08:24:14.0925480Z Pushing D:\a\1\s\bin\nuget\Akka.Cluster.Tools.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:14.7313905Z Pushing Akka.Cluster.Tools.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:14.7928936Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:15.7020123Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:15.7028495Z   Created https://www.nuget.org/api/v2/package/ 909ms
2019-09-23T08:24:15.7043619Z Your package was pushed.
2019-09-23T08:24:15.7231864Z Pushing D:\a\1\s\bin\nuget\Akka.DI.Core.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:16.3585182Z Pushing Akka.DI.Core.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:16.4170989Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:17.1419553Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:17.1429688Z   Created https://www.nuget.org/api/v2/package/ 725ms
2019-09-23T08:24:17.1447739Z Your package was pushed.
2019-09-23T08:24:17.1636679Z Pushing D:\a\1\s\bin\nuget\Akka.DI.TestKit.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:17.7593466Z Pushing Akka.DI.TestKit.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:17.8161351Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:18.5717603Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:18.5726529Z   Created https://www.nuget.org/api/v2/package/ 756ms
2019-09-23T08:24:18.5742455Z Your package was pushed.
2019-09-23T08:24:18.5932467Z Pushing D:\a\1\s\bin\nuget\Akka.DistributedData.1.3.15-beta.nupkg Attempts left: 3
2019-09-23T08:24:19.2271974Z Pushing Akka.DistributedData.1.3.15-beta.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:19.2848805Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:20.0292655Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:20.0330409Z   Created https://www.nuget.org/api/v2/package/ 738ms
2019-09-23T08:24:20.0330938Z Your package was pushed.
2019-09-23T08:24:20.0530883Z Pushing D:\a\1\s\bin\nuget\Akka.DistributedData.LightningDB.1.3.15-beta.nupkg Attempts left: 3
2019-09-23T08:24:20.6627485Z Pushing Akka.DistributedData.LightningDB.1.3.15-beta.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:20.7190582Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:21.3003947Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:21.3014153Z   Created https://www.nuget.org/api/v2/package/ 582ms
2019-09-23T08:24:21.3030768Z Your package was pushed.
2019-09-23T08:24:21.3229786Z Pushing D:\a\1\s\bin\nuget\Akka.FSharp.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:21.9286739Z Pushing Akka.FSharp.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:21.9858450Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:22.8532109Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:22.8541156Z   Created https://www.nuget.org/api/v2/package/ 868ms
2019-09-23T08:24:22.8556544Z Your package was pushed.
2019-09-23T08:24:22.8820831Z Pushing D:\a\1\s\bin\nuget\Akka.MultiNodeTestRunner.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:23.4957657Z Pushing Akka.MultiNodeTestRunner.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:23.5562744Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:27.8258587Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:27.8269205Z   Created https://www.nuget.org/api/v2/package/ 4270ms
2019-09-23T08:24:27.8286027Z Your package was pushed.
2019-09-23T08:24:27.8508193Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:28.7945780Z Pushing Akka.Persistence.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:28.7947570Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:29.5154716Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:29.5163379Z   Created https://www.nuget.org/api/v2/package/ 977ms
2019-09-23T08:24:29.5178892Z Your package was pushed.
2019-09-23T08:24:29.5363949Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.FSharp.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:30.1616510Z Pushing Akka.Persistence.FSharp.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:30.2204141Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:31.2872526Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:31.2874893Z   Created https://www.nuget.org/api/v2/package/ 1066ms
2019-09-23T08:24:31.2882889Z Your package was pushed.
2019-09-23T08:24:31.3075445Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.Query.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:31.9374926Z Pushing Akka.Persistence.Query.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:31.9976843Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:32.6671130Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:32.6679991Z   Created https://www.nuget.org/api/v2/package/ 670ms
2019-09-23T08:24:32.6695196Z Your package was pushed.
2019-09-23T08:24:32.6873850Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.Query.Sql.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:33.3163958Z Pushing Akka.Persistence.Query.Sql.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:33.4949519Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:34.5197558Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:34.5205351Z   Created https://www.nuget.org/api/v2/package/ 1025ms
2019-09-23T08:24:34.5227302Z Your package was pushed.
2019-09-23T08:24:34.5410520Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.Sql.Common.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:35.1319665Z Pushing Akka.Persistence.Sql.Common.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:35.1872426Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:35.9147830Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:35.9156398Z   Created https://www.nuget.org/api/v2/package/ 728ms
2019-09-23T08:24:35.9169892Z Your package was pushed.
2019-09-23T08:24:35.9353347Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.Sql.TestKit.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:36.5405405Z Pushing Akka.Persistence.Sql.TestKit.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:36.6003918Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:37.4559588Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:37.4568480Z   Created https://www.nuget.org/api/v2/package/ 856ms
2019-09-23T08:24:37.4582882Z Your package was pushed.
2019-09-23T08:24:37.4769281Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.Sqlite.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:38.0974519Z Pushing Akka.Persistence.Sqlite.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:38.1543824Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:38.9181415Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:38.9222160Z   Created https://www.nuget.org/api/v2/package/ 760ms
2019-09-23T08:24:38.9223160Z Your package was pushed.
2019-09-23T08:24:38.9364718Z Pushing D:\a\1\s\bin\nuget\Akka.Persistence.TCK.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:39.5466860Z Pushing Akka.Persistence.TCK.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:39.6067372Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:40.5141299Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:40.5150912Z   Created https://www.nuget.org/api/v2/package/ 908ms
2019-09-23T08:24:40.5167095Z Your package was pushed.
2019-09-23T08:24:40.5348435Z Pushing D:\a\1\s\bin\nuget\Akka.Remote.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:41.1604384Z Pushing Akka.Remote.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:41.2172643Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:42.1335232Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:42.1345043Z   Created https://www.nuget.org/api/v2/package/ 917ms
2019-09-23T08:24:42.1360163Z Your package was pushed.
2019-09-23T08:24:42.1551465Z Pushing D:\a\1\s\bin\nuget\Akka.Remote.TestKit.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:42.7865006Z Pushing Akka.Remote.TestKit.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:42.8459402Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:43.6714666Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:43.6724876Z   Created https://www.nuget.org/api/v2/package/ 826ms
2019-09-23T08:24:43.6741322Z Your package was pushed.
2019-09-23T08:24:43.6949905Z Pushing D:\a\1\s\bin\nuget\Akka.Remote.Transport.Helios.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:44.3185773Z Pushing Akka.Remote.Transport.Helios.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:44.3795256Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:45.0266864Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:45.0276521Z   Created https://www.nuget.org/api/v2/package/ 648ms
2019-09-23T08:24:45.0292638Z Your package was pushed.
2019-09-23T08:24:45.0486810Z Pushing D:\a\1\s\bin\nuget\Akka.Serialization.Hyperion.1.3.15-beta.nupkg Attempts left: 3
2019-09-23T08:24:45.6824623Z Pushing Akka.Serialization.Hyperion.1.3.15-beta.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:45.7376846Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:46.3194308Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:46.3203837Z   Created https://www.nuget.org/api/v2/package/ 582ms
2019-09-23T08:24:46.3219654Z Your package was pushed.
2019-09-23T08:24:46.3410346Z Pushing D:\a\1\s\bin\nuget\Akka.Serialization.TestKit.1.3.15-beta.nupkg Attempts left: 3
2019-09-23T08:24:46.9651654Z Pushing Akka.Serialization.TestKit.1.3.15-beta.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:47.0253813Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:47.6756696Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:47.6765245Z   Created https://www.nuget.org/api/v2/package/ 651ms
2019-09-23T08:24:47.6779175Z Your package was pushed.
2019-09-23T08:24:47.6975112Z Pushing D:\a\1\s\bin\nuget\Akka.Streams.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:48.3150032Z Pushing Akka.Streams.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:48.3754158Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:49.4563170Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:49.4572654Z   Created https://www.nuget.org/api/v2/package/ 1081ms
2019-09-23T08:24:49.4588632Z Your package was pushed.
2019-09-23T08:24:49.4772722Z Pushing D:\a\1\s\bin\nuget\Akka.Streams.TestKit.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:50.0972524Z Pushing Akka.Streams.TestKit.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:50.1578560Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:50.8533712Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:50.8544841Z   Created https://www.nuget.org/api/v2/package/ 696ms
2019-09-23T08:24:50.8560636Z Your package was pushed.
2019-09-23T08:24:50.8742560Z Pushing D:\a\1\s\bin\nuget\Akka.TestKit.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:51.4931292Z Pushing Akka.TestKit.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:51.5556052Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:52.3710885Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:52.3740312Z   Created https://www.nuget.org/api/v2/package/ 808ms
2019-09-23T08:24:52.3740726Z Your package was pushed.
2019-09-23T08:24:52.3940928Z Pushing D:\a\1\s\bin\nuget\Akka.TestKit.Xunit.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:52.9935404Z Pushing Akka.TestKit.Xunit.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:53.0542254Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:53.8191818Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:53.8200565Z   Created https://www.nuget.org/api/v2/package/ 765ms
2019-09-23T08:24:53.8216187Z Your package was pushed.
2019-09-23T08:24:53.8417115Z Pushing D:\a\1\s\bin\nuget\Akka.TestKit.Xunit2.1.3.15.nupkg Attempts left: 3
2019-09-23T08:24:54.4717985Z Pushing Akka.TestKit.Xunit2.1.3.15.nupkg to 'https://www.nuget.org/api/v2/package'...
2019-09-23T08:24:54.5268871Z   PUT https://www.nuget.org/api/v2/package/
2019-09-23T08:24:55.2740124Z WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl.
2019-09-23T08:24:55.2748682Z   Created https://www.nuget.org/api/v2/package/ 747ms
2019-09-23T08:24:55.2762032Z Your package was pushed.
2019-09-23T08:24:55.2943195Z Finished Target: PublishNuget
```

--- a/build.fsx
+++ b/build.fsx
@@ -471,7 +471,7 @@ Target "CreateMntrNuget" (fun _ ->
 
 Target "PublishNuget" (fun _ ->
     let nugetExe = FullName @"./tools/nuget.exe"
-    let rec publishPackage url accessKey trialsLeft packageFile =
+    let rec publishPackage url apiKey trialsLeft packageFile =
         let tracing = enableProcessTracing
         enableProcessTracing <- false
         let args p =
@@ -481,33 +481,23 @@ Target "PublishNuget" (fun _ ->
 
         tracefn "Pushing %s Attempts left: %d" (FullName packageFile) trialsLeft
         try 
-            let result = ExecProcess (fun info -> 
-                    info.FileName <- nugetExe
-                    info.WorkingDirectory <- (Path.GetDirectoryName (FullName packageFile))
-                    info.Arguments <- args (packageFile, accessKey,url)) (System.TimeSpan.FromMinutes 1.0)
-            enableProcessTracing <- tracing
-            if result <> 0 then failwithf "Error during NuGet symbol push. %s %s" nugetExe (args (packageFile, "key omitted",url))
+            DotNetCli.RunCommand
+                (fun p -> 
+                    { p with 
+                        TimeOut = TimeSpan.FromMinutes 10. })
+                (sprintf "nuget push %s --api-key %s --source %s" packageFile apiKey url)
         with exn -> 
-            if (trialsLeft > 0) then (publishPackage url accessKey (trialsLeft-1) packageFile)
+            if (trialsLeft > 0) then (publishPackage url apiKey (trialsLeft-1) packageFile)
             else raise exn
     let shouldPushNugetPackages = hasBuildParam "nugetkey"
-    let shouldPushSymbolsPackages = (hasBuildParam "symbolspublishurl") && (hasBuildParam "symbolskey")
     
-    if (shouldPushNugetPackages || shouldPushSymbolsPackages) then
+    if (shouldPushNugetPackages) then
         printfn "Pushing nuget packages"
         if shouldPushNugetPackages then
             let normalPackages= !! (outputNuGet @@ "*.nupkg") |> Seq.sortBy(fun x -> x.ToLower())
             for package in normalPackages do
                 try
                     publishPackage (getBuildParamOrDefault "nugetpublishurl" "https://api.nuget.org/v3/index.json") (getBuildParam "nugetkey") 3 package
-                with exn ->
-                    printfn "%s" exn.Message
-
-        if shouldPushSymbolsPackages then
-            let symbolPackages= !! (outputNuGet @@ "*.snupkg") |> Seq.sortBy(fun x -> x.ToLower())
-            for package in symbolPackages do
-                try
-                    publishPackage (getBuildParam "symbolspublishurl") (getBuildParam "symbolskey") 3 package
                 with exn ->
                     printfn "%s" exn.Message
 )

